dbtmetabase/dbt.py

```diff
@@ -36,12 +36,23 @@ def read_models(self, includes=[], excludes=[]) -> list:
         mb_models = []
 
         for path in (Path(self.project_path) / "models").rglob("*.yml"):
+            logging.info(path)
```

Comment:
Explain what is being done with the path in the log:

```
logging.info("Processing model: %s", path)
```

Reply:
Added as suggested, thanks much better